ethereum/jsonrpc/src/main/java/tech/pegasys/pantheon/ethereum/jsonrpc/internal/methods/NetServices.java

```diff
@@ -20,10 +20,8 @@
 import tech.pegasys.pantheon.ethereum.p2p.api.P2PNetwork;
 import tech.pegasys.pantheon.metrics.prometheus.MetricsConfiguration;
 
-import java.util.LinkedHashMap;
-import java.util.Map;
-
 import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.ImmutableMap.Builder;
```

Comment:
This causes ErrorProne to throw an error.  Please use ImmutableMap.Builder instead of importing the inner class.

https://jenkins.pegasys.tech/job/Pantheon/job/PR-1295/2/execution/node/59/log/
https://errorprone.info/bugpattern/BadImport

Reply:
Thanks. I have changed it and performed these gradle task aswell locally; (spotlessApply clean check test). I hope there are no more errors :)